admin-site/src/App.svelte

```diff
@@ -0,0 +1,421 @@
+										<h3>{event.label} - Occasion # {occasion.id} </h3>
+											
+									</button>
+								{/each}
+							{/if}
+					{/each}
+		{/if}
+	</section>
+</div>
+
+
+
+<div id = "openEvent">
+	<div class="container-fluid">
+			<div class="row">
+				<h5> {focusedEvent.label} - {formattedStartTime}</h5>
+			</div>
+
+			<!-- <div class="row">
+				<div class="col-md-3"> -->
+					<button type="button" class="btn btn-primary" value = "openEvent" on:click={backToOccasionList}>Back To Occasion List</button>
+				<!-- </div>
+			</div> -->
+			<hr>
+
+			<div class="row">
+				<div class="col-md-12">
+					<button type="button" class="btn btn-danger btn-block" on:click={confirmEnd}>End Occasion</button>
+				</div>
+			</div>
+
+			<div class="row">
+				<div class="col-md-12">
+					<button type="button" class="btn btn-primary btn-block" value="openEvent" on:click={showQR}><u>Show QR Code</u></button>
+				</div>
+			</div>
+
+			<div class="row">
+				<div class="col-md-12">
+					<h5>Cue Details</h5>
+					{#if focusedEvent != "undefined"}
+						{#each focusedEvent.cues as cue }
+							{#if cue.cueNumber == cueState}
+								<div id = "{cue.cueNumber}">
+									<ul>Media Domain: {cue.mediaDomain}</ul>
+									<ul>Cue Number: {cue.cueNumber}</ul>
+									<ul>Cue Action: {cue.cueAction}</ul>
+								</div>
+							{/if}
+						{/each}
+					{/if}
+					
+				</div>
+			</div>
+
+			<div class="row">
+				<div class="col-md-6">
+					<button type="button" class="btn btn-info btn-block" value="previous" on:click={changeCueState}>
+						<span class="fas fa-angle-left"></span>
+						Previous
+					</button>
+				</div>
+				<div class="col-md-6">
+					<button type="button" class="btn btn-info btn-block" value="next" on:click={changeCueState}>
+						Next
+						<span class="fas fa-angle-right"></span>
+					</button>
+				</div>
+			</div> 
+
+			<div class="row">
+				<div class="col-md-12">
+					<Slider/>
+				</div>
+			</div>
+	</div>
+</div>
+
+<div id = "QRcode">
+	<div class="modal-dialog modal-lg">
+		<div class="modal-content">
+			<div class="modal-header">
+				<button type="button" class="close" data-dismiss="modal" aria-label="Close" value= "QRcode" on:click={backToOccasionList}>
+					<span aria-hidden="true">&times;</span>
+				</button>
+			</div>
+			<div class="modal-body">
+				<div class="container-fluid">
+					<div class="row">
+						<div class="col-md-12 text-center">
+							<img src="QrCodes/Event1.png " class="img-fluid" alt="QR Code for">
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
+
+<div id = "closeEvent">
+	<div class="container-fluid">
+			<div class="row">
+				<div class="col-md-12">
+        			<h5 class="modal-title">{focusedEvent.label} - {formattedStartTime}.</h5>
+				</div>
+			</div>
+			<div class="row">
+				<button type='button' class="btn btn-primary" value="closeEvent" on:click={backToOccasionList}>Back To Occasions List</button>
```

Comment:
See if you can do a similar adjustment here: back button with text '< Back' on same line as title? For the title I like your idea, but use a shorter date format. 'LOT X - May 24'

Reply:
Change applied, but kept the year as well.